lib/gitx/github.rb

```diff
@@ -103,12 +103,12 @@ def create_authorization
       password = ask_without_echo("Github password for #{username}: ")
       client = Octokit::Client.new(login: username, password: password)
       options = {
-        :scopes => ['repo'],
-        :note => github_client_name,
-        :note_url => CLIENT_URL
+        scopes: ['repo'],
+        note: github_client_name,
+        note_url: CLIENT_URL
       }
       two_factor_auth_token = ask_without_echo('Github two factor authorization token (if enabled): ')
-      options[:headers] = {'X-GitHub-OTP' => two_factor_auth_token} if two_factor_auth_token
+      options[:headers] = { 'X-GitHub-OTP' => two_factor_auth_token } if two_factor_auth_token
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.